LibreNMS/OS/Zyxelnwa.php

```diff
@@ -23,12 +23,37 @@
 
 class Zyxelnwa extends Zyxel implements OSDiscovery, WirelessClientsDiscovery
 {
+    
     public function discoverWirelessClients()
     {
-        $oid = '.1.3.6.1.4.1.890.1.15.3.5.1.1.2.1'; //ZYXEL-ES-SMI::esMgmt.5.1.1.2.1
+        $sensors = [];
+        $data = $this->getCacheTable('ZYXEL-ES-WIRELESS::wlanRadioTable');
```

Comment:
No need to cache this, it is only used here, right?

Just use SnmpQuery::walk()